assert/assertions_test.go

```diff
@@ -2919,7 +2702,7 @@ func TestErrorAs(t *testing.T) {
 
 func TestIsNil(t *testing.T) {
```

Comment:
This test should also move to `internal/check`.

Reply:
Turns out this can actually just be deleted.